internal/structured_logger_test.go

```
@@ -190,3 +191,77 @@ func TestStructuredLoggerWithDefaultStatus(t *testing.T) {
+			logHealth: "false",
+			shouldLog: true,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			// Save current env and restore after test
+			oldLogHealth, exists := os.LookupEnv("LOG_HEALTH")
+			defer func() {
+				if exists {
+					os.Setenv("LOG_HEALTH", oldLogHealth)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)